lib/maths/unittest/CTreeShapFeatureImportanceTest.cc

```diff
@@ -107,9 +107,9 @@ struct SFixtureSingleTree {
 
         auto& tree = s_Trees[0];
         tree.resize(1);
-        tree[0].split(0, 0.5, true, 0.0, 0.0, tree);
-        tree[1].split(1, 0.5, true, 0.0, 0.0, tree);
-        tree[2].split(1, 0.5, true, 0.0, 0.0, tree);
+        tree[0].split({}, 0, 0.5, true, 0.0, 0.0, tree);
```

Comment:
Would you please add a `/*comment*/` about what `{}` suppose to be and is missing to improve readability?

Reply:
I ended up providing an overload of split which supplies empty and explain in its documentation. See [3720796](https://github.com/elastic/ml-cpp/pull/2031/commits/372079695a3cff2cc7e5440628e0480d1a5a0e41).